docs/topics/installing-plugin.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
```

Comment:
Why is this module among your changes? It does not appear to be related to the VS Code extension guide.

Reply:
I think it is because originally I added ifdef sections to this module with the attribute for the VS Code extension. Doing that worked fine but at one point you and I discussed it and I decided to create a separate installation module for the VS Code extension. This "added" module has been part of the PR for a while. If you think it's wiser, I can close this PR and open a new one.

Reply:
That should not happen. This PR records the file as a new file, not a changed file. Check it carefully. I would not close the PR and open a new one. But I would make sure that this is not duplicating an existing file.

Reply:
The file is irrelevant, so i deleted it in the current commit. Eclipse/CSR installation is covered by eclipse-installing-plugin.adoc and VS Code installation is covered by vs-code-extension.adoc.